tests/ops/qubit/test_qubit_ops.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the available built-in discrete-variable quantum operations. Only tests over multiple
+types of operations should exist in this file. Type-specific tests should go in the more specific file.
+"""
+import pytest
+import pennylane as qml
+
+from gate_data import I
+
+
+class TestOperations:
```

Comment:
@albi3ro wondering about this file, would the content here belong into the parametric test file?

Reply:
Because it's also testing non-parametrized gates as well.  We could just split the test in two...

Reply:
Yep, I think that could work. It feels a bit strange to have this file even after the split of files.